_episodes/01-introduction.md

```diff
@@ -217,6 +217,21 @@ Here are a few example for combinations of these.
 '--'   # dashed line with default color
 '^k:'  # black triangle_up markers connected by a dotted line
 ```
+# Exercise:
+
+### Exercise: Using the table above, plot functions using different linestyles/markers on the same canvas:
+> ## Solution
+> ```python
+> x1 = np.linspace(1, 10)
+> y1, y2, y3 = np.log(x1), np.cos(x1), np.sin(x1)
+> fig, ax = plt.subplots()  # Create a figure containing a single axes.
+> ax.plot(x1, y1, linestyle="--", label="Dashed line")  # Create
+> ax.plot(x1, y2, linestyle=":", label="Dotted line style")
+> ax.plot(x1, y3, marker="o", label="Circle marker")
+> ax.legend()
+> plt.show()
```

Comment:
This should be `fit.show()` with the OO API

Reply:
Hi Thanks for your comments. I have implemented both the comments.